src/main/kotlin/no/nav/syfo/application/IBehandlerMeldingProducer.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.syfo.application
+
+import no.nav.syfo.domain.BehandlerMelding
+import no.nav.syfo.domain.Personident
+
+interface IBehandlerMeldingProducer {
+    fun send(
+        personident: Personident,
+        behandlermelding: BehandlerMelding,
```

Comment:
Kan vi kalle det `behandlermelding` overalt, i stedet for `behandlerMelding`? 🤞 Eventuelt om vi har veldig lyst til å skrive det i to ord, at vi også gjør det på denne og `behandlermeldingPdf`, sånn at det blir likt.

Reply:
Ja, jeg tenker det hadde vært fint med domeneklassen også 🥳

Reply:
Det var vel jeg som syndet på tabell-navnet in the first place 😢 Men enig i consistency på `Behandlermelding` i koden så langt det lar seg gjøre!